shared/versions.yml

```diff
@@ -56,7 +56,7 @@ prometheus_to_sd:
 couchdb:
   upstream:
     repository: couchdb
-    tag: 2.2.0
+    tag: 2.3.0
```

Comment:
Is there a reason not to use the latest 2.3.1?

Reply:
Not sure how I missed that version (release a couple weeks ago), but I'll stick with the version I've tested.